engine/controllers/server.cc

```diff
@@ -95,15 +96,15 @@ void server::GetModels(const HttpRequestPtr& req,
 void server::GetEngines(
     const HttpRequestPtr& req,
     std::function<void(const HttpResponsePtr&)>&& callback) {
-  auto ir = inference_svc_.GetEngines(req->getJsonObject());
+  auto ir = inference_svc_->GetEngines(req->getJsonObject());
   auto resp = cortex_utils::CreateCortexHttpJsonResponse(ir);
   callback(resp);
 }
 
 void server::FineTuning(
     const HttpRequestPtr& req,
     std::function<void(const HttpResponsePtr&)>&& callback) {
-  auto ir = inference_svc_.FineTuning(req->getJsonObject());
+  auto ir = inference_svc_->FineTuning(req->getJsonObject());
```

Comment:
@vansangpfiev Can I check: why do we have a FineTuning endpoint, if we haven't implemented it yet?

Reply:
This is the legacy code from last time when we were trying to integrate `cortex.python` into `cortex.cpp`